src/multiform/MultiForm.ts

```diff
@@ -27,6 +27,7 @@ export default class MultiForm extends AVisInstance implements IVisInstance, IMu
    * list of all possibles vis techniques
    */
   readonly visses: IVisPluginDesc[];
+  brushed: boolean = false;
```

Comment:
@bikramkawan This shouldn't be necessary, you have it in your extended class, right?